packages/crypto/src/p256/operations.ts

```diff
@@ -46,8 +46,8 @@ export const verify = async (
   return webcrypto.subtle.verify(
     { name: 'ECDSA', hash: { name: 'SHA-256' } },
     importedKey,
-    sig,
-    data,
+    new Uint8Array(sig),
+    new Uint8Array(data),
```

Comment:
Not sure I totally get why these are needed. Or is the idea to be a bit defensive here?

Reply:
Yeah just being extra defensive. it's probably not necessary. But the different types of binary representations freak me out enough that I just decided to wrap it